src/components/stack-chart/Canvas.tsx

```diff
@@ -638,6 +650,7 @@ class StackChartCanvasImpl extends React.PureComponent<Props> {
         callTreeSummaryStrategy="timing"
         durationText={durationText}
         displayStackType={displayStackType}
+        argumentValues={argumentSummaries}
```

Comment:
It would be good to be consistent with the naming here. Either we should use `argumentValues` or `argumentSummaries`. I'm leaning towards `argumentSummaries` since this is the return value of `getArgumentSummaries`